grammar.js

```diff
@@ -321,14 +322,13 @@ module.exports = grammar({
 
     chunk: ($) => /[^\n]+/,
 
-    _script_line: ($) => seq(optional($.chunk), '\n'),
+    _script_line: ($) => seq(optional(alias($.chunk, 'chunk')), '\n'),
```

Comment:
I thought it was better to make each individual line an anonymous node rather than a named one to make the generated AST lighter, but I guess I can undo the change when I get home.

Reply:
No but as it is right now, this is redundant:
```suggestion
    _script_line: ($) => seq(optional($.chunk), '\n'),
```

Reply:
Done, I have rebased it onto the first diverging commit of the branch and made the necessary changes in the `test/corpus` dir.